tests/phpunit/Presentation/SidebarHtmlBuilderIntegrationTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace ProfessionalWiki\WikibaseFacetedSearch\Tests\Presentation;
+
+use MediaWiki\MediaWikiServices;
+use MediaWikiIntegrationTestCase;
+use ProfessionalWiki\WikibaseFacetedSearch\Presentation\SidebarHtmlBuilder;
+use ProfessionalWiki\WikibaseFacetedSearch\WikibaseFacetedSearchExtension;
+
+/**
+ * @covers \ProfessionalWiki\WikibaseFacetedSearch\Presentation\SidebarHtmlBuilder
+ * @covers \ProfessionalWiki\WikibaseFacetedSearch\WikibaseFacetedSearchExtension
+ * @group Database
+ */
+class SidebarHtmlBuilderIntegrationTest extends MediaWikiIntegrationTestCase {
+
+	public function testIntegrationSmoke(): void {
+		$this->overrideConfigValue(
+			WikibaseFacetedSearchExtension::CONFIG_VARIABLE_NAME,
+			<<<JSON
+{
+	"itemTypeProperty": "P42",
+	"configPerItemType": {
+		"Q1": {
+			"label": "People",
+			"facets": {
+				"P100": {
+					"type": "list"
+				}
+			}
+		}
+	}
+}
+JSON
+		);
+
+		$html = $this->getSidebarHtmlBuilderFromGlobals()->createHtml( 'foo haswbfacet:P42=Q1' );
+		$this->assertStringContainsString( 'sidebar', $html );
```

Comment:
The original test did not have this check. Added it here by updating the test config and query.